services/proctoring-service/src/controllers/proctoring.controller.js

```
@@ -72,9 +72,81 @@ async function getStudentsInExam(req, res) {
+
+// <<< BỔ SUNG >>>
+/**
+ * Controller để lấy tất cả sinh viên đang thi trong một kỳ thi.
+ */
+async function getStudentsInExam(req, res) {
+  try {
+    const { examId } = req.params;
+    if (!examId) {
+      return res.status(400).json({ message: 'Exam ID is required.' });
+    }
+    const students = await proctoringService.getStudentsInExam(examId);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'getStudentsInExam'. Consider to delete it or rename it.

'getStudentsInExam' is defined here:


(lint/suspicious/noRedeclare)